mesonbuild/modules/gnome.py

```diff
@@ -317,6 +319,21 @@ def _print_gdbus_warning() -> None:
                      mlog.bold('https://github.com/mesonbuild/meson/issues/1387'),
                      once=True, fatal=False)
 
+    def _find_tool(self, state: 'ModuleState', tool: str) -> 'ToolType':
+        tool_map = {
+            'gio-querymodules': 'gio-2.0',
+            'glib-compile-schemas': 'gio-2.0',
+            'glib-compile-resources': 'gio-2.0',
+            'gdbus-codegen': 'gio-2.0',
+            'glib-genmarshal': 'glib-2.0',
+            'glib-mkenums': 'glib-2.0',
+            'g-ir-scanner': 'gobject-introspection-1.0',
+            'g-ir-compiler': 'gobject-introspection-1.0',
+        }
+        pcfile = tool_map[tool]
```

Comment:
I believe they are called modules in pkgconfig, right? Not files.

Reply:
Renamed to depname which is what's used for state.find_tool() argument as well.